packages/doenetml-worker-rust/lib-doenetml-core/src/core/document_renderer/to_flat_dast.rs

```diff
@@ -9,13 +10,21 @@ use crate::{
     core::document_model::DocumentModel,
     dast::{
         flat_dast::UntaggedContent, DastAttribute, DastText, DastTextRefContent,
-        FlatDastElementUpdate, FlatDastRoot, ForRenderPropValue, ForRenderProps,
+        FlatDastElementUpdate, FlatDastRoot, ForRenderPropValue, ForRenderPropValueOrContent,
+        ForRenderProps,
     },
     graph::directed_graph::Taggable,
     props::{cache::PropWithMeta, PropProfile, PropValue},
+    state::types::content_refs::ContentRef,
 };
 
-use super::{super::graph_node::GraphNode, DocumentRenderer};
+/// When to include a prop in the serialized rendered props.
+pub enum SerializeCondition {
```

Comment:
Don't want to get burned anymore by misreading a boolean condition?

Reply:
You know me so well!